modules/ROOT/pages/install-self-managed.adoc

```diff
@@ -0,0 +1,85 @@
+= Install Runtime Fabric on Self-managed Kubernetes
+
+You can install Anypoint Runtime Fabric into a Kubernetes cluster provided by AWS Elastic Kubernetes Service (EKS) or Azure Kubernetes Service (AKS). 
+
+== Before you Begin
+
+Before installing Anypoint Runtime Fabric in a self-managed Kubernetes service, ensure the following:
+
+* You have installed and configured your Kubernetes cluster as follows:
++
+- Running Kubernetes version 1.14.x to 1.18.x.
+- Running an ingress controller to send external requests to deployments.
+* You have downloaded and installed the `rtfctl` command-line utility.
++
+`rtfctl` is supported on Windows, MacOS (Darwin), and Linux. Download this utility using the URLs below:
++
+*Windows:*
++
+----
+curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl-windows/latest -o rtfctl
+----
++
+*MacOS (Darwin):*
++
+----
+curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl-darwin/latest -o rtfctl
+----
++
+*Linux:*
++
+----
+curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -o rtfctl
+----
+
+== Create a Runtime Fabric using Runtime Manager
+
+To install Runtime Fabric on a self-managed Kubernetes cluster, first create a Runtime Fabric using Runtime Manager. This is required to obtain the activation data which is needed during installation.
+
+. From Anypoint Platform, select Runtime Manager.
+. Click *Runtime Fabrics*.
+. Click *Create Runtime Fabric*.
+. Enter the name of the new Runtime Fabric, then select one of the following options:
++
+* Amazon Elastic Kubernetes Service
+* Azure Kubernetes Service
+
+. Click *Next*.
+. Review the *Support responsibility* disclaimer, then if you agree click *Continue*.
++
+Runtime Manager creates the Runtime Fabric and displays the Activation State page. This page displays the activation data used to install Runtime Fabric on a Kubernetes service. Copy this data to the clipboard for use in the next section. 
+
+
+== Install Runtime Fabric on a Self-Managed Kubernetes Cluster
+
+After creating a Runtime Fabric and getting the activation data, install Runtime Fabric into your Kubernetes service using the `rtfctl` command.
+
+[NOTE]
+====
+Run `rtfctl --help` to see a list of options for this command.
+====
+
+. From the command line, run `rtfctl --validate`.
```

Comment:
Correction: `rtfctl validate`

Reply:
The -- is still required though, no? I thought in the demo that was what was used.

Reply:
Nevermind. I just verified. There is no -- in the demo.

Reply:
Fixed.